packages/slice-machine/server/src/api/custom-types/push.ts

```diff
@@ -26,7 +26,7 @@ const createOrUpdate = (
 export default async function handler(query: { id: string }) {
   const { id } = query;
 
-  const state = await fetchState();
+  const state = await getBackendState();
```

Comment:
like the renaming, makes it much clearer

Reply:
Good